### ISSUE TITLE
`dirtyChecking` consumes too much CPU performance in viewer.js when idle

### ISSUE THREAD

**Issue by purocean** [jgraph/drawio#2766]
### Preflight Checklist

* [x] I agree to follow the [Code of Conduct](https://github.com/jgraph/drawio/blob/master/CODE_OF_CONDUCT.md) that this project adheres to.
* [x] I have searched the issue tracker for a feature request that matches the one I want to file, without success.

**Describe the bug**
`dirtyChecking` consumes too much CPU performance in viewer.js when idle

**To Reproduce**

https://codesandbox.io/s/hidden-tdd-eeo0b3?file=/public/index.html

Open chorme devtools.

Do nothing, watch performance panel.

**Expected behavior**

Do not call dirtyChecking frequently.

**Screenshots**
![image](https://user-images.githubusercontent.com/7115690/163954222-7c2063d7-eb46-4c0d-a326-abe36d0c2ed2.png)
![image](https://user-images.githubusercontent.com/7115690/163954360-ddc8fc10-f661-4a89-b54b-26a366a478cb.png)
![image](https://user-images.githubusercontent.com/7115690/163954841-76f91ebb-06e5-4e8d-917e-f09fdc00a2ac.png)

https://github.com/jgraph/drawio/blob/82025f7f452d8bffcec468a5542a3bc6f8631ae4/src/main/webapp/js/diagramly/GraphViewer.js#L2413

**draw.io version (In the Help->About menu of the draw.io editor):**

- draw.io version 17.4.3

**Desktop (please complete the following information):**
 - OS: macOS
 - Browser Chrome
 - Version 100

**Comment by purocean** [comment]
I use `viewer.min.js` in iframe, a temporary solution

```js
window._requestAnimationFrameCount = 0
window._requestAnimationFrame = window.requestAnimationFrame
window._requestAnimationFrameTask = null

window.addEventListener('mousemove', () => {
  window._requestAnimationFrameCount = 0
  if (window._requestAnimationFrameTask) {
    window._requestAnimationFrameTask()
    window._requestAnimationFrameTask = null
  }
})

window.requestAnimationFrame = function (fn) {
  if (window._requestAnimationFrameCount < 200) {
    window._requestAnimationFrameCount++
    window._requestAnimationFrameTask = null
    return window._requestAnimationFrame(fn)
  } else {
    window._requestAnimationFrameTask = fn
  }
}
```

**Comment by davidjgraph** [comment]
Is the issue that the CPU load is high, or that the function is called frequently? Being called frequently, but doing nothing on the vast majority of calls doesn't imply high CPU. Where is the cumlative CPU load for that function?

**Comment by purocean** [comment]
I embed drawio graphics into my Electron app. There is user feedback that when doing nothing, the program also has about 5% CPU usage. So I checked here.

**Comment by purocean** [comment]
> I use `viewer.min.js` in iframe, a temporary solution
> 
> 
> 
> ```js
> 
> window._requestAnimationFrameCount = 0
> 
> window._requestAnimationFrame = window.requestAnimationFrame
> 
> window._requestAnimationFrameTask = null
> 
> 
> 
> window.addEventListener('mousemove', () => {
> 
>   window._requestAnimationFrameCount = 0
> 
>   if (window._requestAnimationFrameTask) {
> 
>     window._requestAnimationFrameTask()
> 
>     window._requestAnimationFrameTask = null
> 
>   }
> 
> })
> 
> 
> 
> window.requestAnimationFrame = function (fn) {
> 
>   if (window._requestAnimationFrameCount < 200) {
> 
>     window._requestAnimationFrameCount++
> 
>     window._requestAnimationFrameTask = null
> 
>     return window._requestAnimationFrame(fn)
> 
>   } else {
> 
>     window._requestAnimationFrameTask = fn
> 
>   }
> 
> }
> 
> ```

I Use this code. It's work. 

I compare them by task manager (chrome and os) and recoding performance carefully. Display  viewer or not. rewrite `requestAnimationFrame ` or not.

Drawio editor has no this issue, only viewer.

**Comment by davidjgraph** [comment]
Released in 21.2.2
